cicd.tf

```diff
@@ -36,7 +36,7 @@ data "aws_iam_policy_document" "cicd_policy" {
     ]
 
     resources = [
-      data.aws_ecr_repository.ecr.arn,
+      var.ecr_repository_arn
```

Comment:
how does value pass into it? should we define new parameter in inputs? `ecr_repository_arn`

Reply:
There is a variable defined on top. Then when you use module, you just pass in the arn as defined on line 2  y `ecr_repository_arn`. There is a separate file for vars, but this one already had one defined for name so I just renamed it.